research-software-directory/RSD-as-a-service
# fix: remove margin of GitLab icon

# Remove GitLab icon margin

Fixes #402

Changes proposed in this pull request:

* Remove GitLab Icon margin so that it will be displayed the same size as the GitHub Logo

How to test:

* `make dev`
* Change an existing repo type to GitLab, or add a new GitLab repository
* Open the corresponding software page
* The icon should now be approximately the same size as a GitHub Logo:
  ![image](https://user-images.githubusercontent.com/14222414/178440987-ea544550-9558-4693-9761-c8a527f3d12b.png)


PR Checklist:
